lua/cmake/utils.lua

```diff
@@ -59,7 +59,7 @@ function utils.run(cmd, args, opts)
         if opts.on_success then
           opts.on_success()
         end
-      elseif opts.quickfix_only_on_error then
+      elseif config.quickfix_only_on_error then
```

Comment:
Why you removed this change?

Reply:
I wasn't should what to do here :D, should I make it `opts.open_quickfix` ?

Reply:
Oh, never mind. You did it right.

Reply:
On second thought I think it should be like this:
```lua
elseif not opts.show_quickfix then
```
Could you check it?

Reply:
That works, too

Reply:
I just thought using `config.quickfix_only_on_error` would make it easier to understand the elseif statement

Reply:
But then this condition would be called if `config.quickfix_only_on_error = true` and `open_quickfix = true`. I.e. quickfix would be opened twice in `cmake.run()`.